File: projects/web/go/gin/{{cookiecutter.project_slug}}/internal/handlers/health.go
// Package handlers is the HTTP layer: parse the request, call a service, write the response.
package handlers

import (
	"net/http"

	"github.com/gin-gonic/gin"

	"github.com/{{ cookiecutter.github_owner }}/{{ cookiecutter.project_slug }}/internal/models"
)

// Health reports the version. Every API exposes it at /ping.
func Health(version string) gin.HandlerFunc {
	return func(c *gin.Context) {
		c.JSON(http.StatusOK, models.Health{Status: "ok", Version: version})
	}
}

File: projects/web/python/fastapi/{{cookiecutter.project_slug}}/app/api/health.py
"""Health. Every API exposes it at the root."""

from fastapi import APIRouter

from app.schemas.health import Health

health_router = APIRouter(tags=["Health"])


@health_router.get("/health")
def health() -> Health:
    from app import __version__

    return Health(status="ok", version=__version__)
